_ptbr/reply-three.md

```diff
@@ -11,11 +11,11 @@ Querido pai,
 
 Sua carta chegou enquanto a mamãe e eu estávamos no hospital. (A vovó Ruža do nosso quarto ficou de olho no Stane durante esse período.)
 
-Fomos ao hospital por comer cogumelos venenosos. A tia Hope, do primeiro andar, me levava até a floresta para colher cogumelos. Era interessante, ela falava sobre a natureza o tempo todo. À noite, voltávamos com cogumelos e todos da sala jantavam juntos. Depois de um jantar desses, eu saí com o Mihajlo. Estávamos vendo a lua pelos binóculos, como se fosse um telescópio. De repente, senti uma fraqueza. Voltei para o quarto e reclamei pra minha mãe que estava doente. Todos se viraram para mim: "Você também?"
+Fomos parar no hospital por comer cogumelos venenosos. Frequentemente a tia Nada, do primeiro andar, me levava para a floresta para colhermos cogumelos. Era interessante, ela falava sobre a natureza o tempo todo. À noite voltávamos com cogumelos e todos do quarto jantavam juntos. Depois de um jantar desses, eu saí com o Mihajlo. Estávamos vendo a lua pelos binóculos, como se fosse um telescópio. De repente, senti uma fraqueza. Voltei para o quarto e contei pra mãe. Todos se viraram para mim: "Você também?"
```

Comment:
```suggestion
Fomos parar no hospital por comer cogumelos venenosos. Frequentemente a tia Nada, do primeiro andar, me levava até a floresta para colhermos cogumelos. Era interessante, ela falava sobre a natureza o tempo todo. À noite voltávamos com cogumelos e todos do quarto jantavam juntos. Depois de um jantar desses, eu saí com o Mihajlo. Estávamos vendo a lua pelos binóculos, como se fosse um telescópio. De repente, senti uma fraqueza. Voltei para o quarto e contei pra mãe. Todos se viraram para mim: "Você também?"
```

Reply:
I disagree on this one: "até" gives me a sense of "until", in which case they'd not enter the forest. I suggest we double check this one as well 🤓

Reply:
Agreed on "na" 👍